tests/test_loky_module.py

```diff
@@ -19,9 +19,13 @@ def test_cpu_count():
     assert type(cpus) is int
     assert cpus >= 1
 
+    cpus_physical = cpu_count(maybe_physical_only=True)
```

Comment:
I do not understand why coverage is low in the new code given that you included that new case. Any idea?

Reply:
Actually, we set `LOKY_MAX_CPU_COUNT` in `.azure_pipeline.yml`. Maybe we should try to stop doing this. It might not be needed anymore.

Reply:
I only see it set for pypy. Maybe the coverage issue is caused by all the failing jobs by not adding their coverage report ?
btw the failures does not seem to be related to this PR, it's already failing on master

Reply:
ah yes offcourse you are right.